tests/test_adapter.py

```diff
@@ -99,7 +99,7 @@ def test_adapter_factory_TestAdapter_invalid_raised():
         get_adapter_factory(settings)
         pytest.fail(msg="Invalid adapter not inheriting from 'AdapterInterface' should raise on import.")
     adapter_str = '{}.{}'.format(AdapterInterface.__module__, AdapterInterface.__name__)
-    assert adapter_str in str(err), "Expected to have full adapter import string in error message."
+    assert adapter_str in str(err.value), "Expected to have full adapter import string in error message."
```

Comment:
@cehbrecht 
I'm curious about what this changes?
I though calling `str()` would print the `TypeError(<some-error>)` representation.

Reply:
Say have changed the `ExceptionInfo`, see release notes:
https://docs.pytest.org/en/latest/changelog.html

Reply:
Ok, I think doesn't cause much concern for this specific test.